specification/recoveryservices/resource-manager/Microsoft.RecoveryServices/stable/2020-02-02/vaults.json

```diff
@@ -1315,8 +1315,8 @@
       "type": "object",
       "description": "A resource identity that is managed by the user of the service.",
       "properties": {
-        "principalId": {
-          "description": "The principal ID of the user-assigned identity.",
+        "objectId": {
```

Comment:
The previous property name was correct.  User assigned contract is defined here:  https://armwiki.azurewebsites.net/authorization/managed_identities/MSIOnboardingUserAssigned.html#resource-definition-response

Reply:
Is there an issue if we name it object id instead of principal id as both of them are ideally the same thing. As this code is already released. Also in the Azure portal we see that its only object id which is shown and not principal Id. So do you see any issue if we keep it object id

Reply:
It was a mistake from my end that I took objectId as the contract here, but now changing it in the code contracts would lead to many issues as it is already stored in catalog under the name object id and will involve changes in multiple services. Thats the reason i was asking if its fine if we can keep it object id as the name

Reply:
Can you please reply to Chris' comment? according to our docs, we use principalId not objectId.

Reply:
Updated